src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
Will you bother to strip counterparty where appropriate here? https://github.com/ripple/ripple-lib/blob/develop/src/api/ledger/parse/payment.js#L38